src/shared/organisms/DataPanel/DataPanel.tsx

```diff
@@ -603,7 +649,6 @@ const DataPanel: React.FC<Props> = ({}) => {
               className="my-data-panel-table"
               rowClassName="my-data-panel-table-row"
               pagination={false}
-              scroll={{ y: 400 }}
```

Comment:
why remove the scroll ?

Reply:
My bad. Thanks for pointing out. FIxed it.